Mathlib/Algebra/Order/Interval/Finset/SuccPred.lean

```diff
@@ -70,13 +70,24 @@ lemma insert_Icc_eq_Icc_add_one_right (h : a ≤ b + 1) :
     insert (b + 1) (Icc a b) = Icc a (b + 1) := by
   simpa [← succ_eq_add_one] using insert_Icc_eq_Icc_succ_right (succ_eq_add_one b ▸ h)
 
-lemma insert_Ico_right_eq_Ico_add_one_right_of_not_isMax (h : a ≤ b) (hb : ¬ IsMax b) :
+lemma insert_Ico_eq_Ico_add_one_right_of_not_isMax (h : a ≤ b) (hb : ¬ IsMax b) :
```

Comment:
Just reminding you that yesterday in person you said you wanted to change this name to begin with `insert_Ico_right_`